containers/Connector/config.ts

```diff
@@ -38,19 +38,19 @@ const supportedChains = [
 		label: 'Optimism Mainnet',
 		rpcUrl: getInfuraRpcURL(NetworkIdByName['mainnet-ovm']),
 	},
-	// Kovan
+	// goerli
 	{
-		id: getChainIdHex(NetworkIdByName.kovan),
+		id: getChainIdHex(NetworkIdByName.goerli),
```

Comment:
The contracts will be there, but since it will be shut down I guess we should move to goerli instead. Sincei ts test networks it really only affects us devs